week_4/quiz-trivia-app/src/app/data/quiz.model.ts

```diff
@@ -0,0 +1,7 @@
+export class Quiz {
```

Comment:
If you need an interface, just create a quiz interface instead of a class.

Reply:
done